internal/pkg/runtime/engines/fakeroot/engine_linux.go

```diff
@@ -0,0 +1,191 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package fakeroot
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+
+	"github.com/opencontainers/runtime-tools/generate"
+
+	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	fakerootUtil "github.com/sylabs/singularity/internal/pkg/fakeroot"
+	"github.com/sylabs/singularity/internal/pkg/runtime/engines/config"
+	"github.com/sylabs/singularity/internal/pkg/runtime/engines/config/starter"
+	fakeroot "github.com/sylabs/singularity/internal/pkg/runtime/engines/fakeroot/config"
```

Comment:
Should this be something like `fakerootconf`?

```suggestion
	fakerootconf "github.com/sylabs/singularity/internal/pkg/runtime/engines/fakeroot/config"
```

Reply:
fixed